scripts/build_vocab.py

```diff
@@ -28,10 +28,6 @@
 from joeynmt.tokenizers import BasicTokenizer
 from joeynmt.vocabulary import sort_and_cut
 
-# Sentencepiece Training Params
-CHARACTER_COVERAGE = {"en": 1.0, "de": 1.0, "fr": 1.0, "ja": 0.995, "zh": 0.995}
```

Comment:
Maybe we can still recommend these defaults somewhere? Maybe in the header of this script, or in the "small" configs?